drivers/hyperv/hyperv.go

```diff
@@ -16,16 +17,23 @@ import (
 	"github.com/docker/machine/libmachine/state"
 )
 
+const (
+	DefaultProtocol = iota
```

Comment:
which is? this is not clear... as what defines the default ?

Reply:
It would depend on the adapter and DHCP setting.
I believe it's the reason why sometimes an IPv6 protocol gets returned by the driver and fails minikube.
Do you have any better naming for this option?

Reply:
it also happens with the Defautl Switch, while this should always AFAIK return a valid IPv4 addres... but this is not the case. In case of failures in thenetworking stack we have seen IPv6 being returned as a link-local.

Reply:
I would not say protocol, but just Default.

Reply:
Sounds good.